eng/pipelines/templates/variables/vmr-build.yml

```diff
@@ -14,7 +14,9 @@ variables:
 
 - name: almaLinuxContainer
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:almalinux-8-source-build
-- name: alpineContainer
+- name: alpineLatestContainer
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.20-withnode
+- name: alpineLatestMinusOneContainer
```

Comment:
I'd prefer a name like `alpinePrevious` instead of `alpineLatestMinusOne`.

Reply:
I struggled with the naming.  I like your suggestion better.  Will update.

Reply:
Fixed in https://github.com/dotnet/sdk/pull/42119/commits/201986f511b3232b4e331d4ac41d81a894532715